.github/workflows/build.yml

```diff
@@ -11,17 +11,18 @@ jobs:
         os: [ubuntu-latest, windows-latest]
 
     steps:
+      - name: Check out code repository source code
+        uses: actions/checkout@v2
```

Comment:
We've found in the past that`actions/checkout` needs to be before `actions/setup-node` to leverage caching.